core/eolearn/tests/test_eodata_merge.py

```diff
@@ -183,4 +183,4 @@ def test_lazy_loading(test_eopatch_path):
     eop = merge_eopatches(eop1, eop2, features=[(FeatureType.MASK, ...)])
     assert isinstance(eop.mask.get("CLM"), np.ndarray)
     assert isinstance(eop1.mask.get("CLM"), np.ndarray)
-    assert isinstance(eop1.mask_timeless.get("LULC"), FeatureIO)
+    assert isinstance(eop1.mask_timeless._get_unloaded("LULC"), FeatureIO)
```

Comment:
the old way was actually 'load only on `[item]` access' but not documented anywhere, which i consider very problematic.
I would say that every access has to load, unless explicitly mentioned not to do that (the way copy does it).
Since we have `copy` which can select features to copy over, i don't see much use in have any other way of non-loading access for the general public.